### PR TITLE
feat(language): add tibetan and tibetan 1k (@lobnor2)

### PR DESCRIPTION
**Pull request by lobnor2**
### Description

add tibetan.json and tibetan_1k.json in the frontend/static/languages
update _groups.json and _list.json

### Checks

- [ ] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [x] Adding a language or a theme?
  - [x] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

Closes #

### CODE REVIEW

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/_groups.json [hunk 1]
```diff
@@ -20,6 +20,10 @@
     "name": "spanish",
     "languages": ["spanish", "spanish_1k", "spanish_10k", "spanish_650k"]
   },
+  {
+    "name": "tibetan",
+    "languages": ["tibetan", "tibetan_1k"]
+  },
   {
     "name": "french",
```
[the remark this answers is not in the document]
Comment (lobnor2) [reply frontend/static/languages/_groups.json]: i have arranged language by popularity. 
Have a look

**Comment by Miodec** [comment]
Hey, you've got loads of duplicate words in the files. Please remove them.

**Comment by lobnor2** [comment]
> Hey, you've got loads of duplicate words in the files. Please remove them (the failing logs will say which ones are duplicates).

I have removed duplicate words.

File: frontend/static/languages/tibetan_1k.json [hunk 2]
```diff
@@ -0,0 +1,1089 @@
+{
+  "name": "tibetan",
```
Comment (Miodec) [inline frontend/static/languages/tibetan_1k.json:2]: This should be `tibetan_1k`

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/tibetan_1k.json (hunk 2 above)
Comment (lobnor2) [reply frontend/static/languages/tibetan_1k.json]: i have changed that

**Comment by Miodec** [comment]
Thanks

**Pull request merged**